### PR TITLE
Added a toggle to show/hide "Ping: <number>"

### PR DESCRIPTION
**Pull request by bakemonozero1**
Hi,

I absolutely love the netplay feature in DuckStation, especially when using it while traveling with a friend and both bringing an ROG Ally X. Since our ping is almost always good (we're on LAN), we wanted to disable or hide the ping notification that's always displayed. I checked Discord but couldn't find any information on whether this was possible, and I noticed that you're quite busy.

So, instead of requesting the feature, I went ahead and implemented it myself. It was just a very minor code change, and I made sure to follow your coding style by implementing it in the same way as the ToggleDesyncNotification option.

This addition is based on the nat-traversal branch, as I noticed that this was the most up-to-date branch according to the release notes.

Thought I made a pull request in case you wanted to add this.

To sum it up:

- Added a define hotkey beneath the ToggleDesyncNotification option
- Created small function to actually toggle the value
- If statement around the "Ping:<#>" message

kind regards

### CODE REVIEW

**Comment by HeatXD** [comment]
Im currently working on rewriting netplay to be more up to date with duckstation master. thank you for the suggestion i will add it the list to include in the next version.
